src/main/java/io/weidongxu/util/releaseautomation/Utils.java

```diff
@@ -136,6 +138,58 @@ public static CommitStatus[] getCommitStatuses(HttpPipeline httpPipeline, String
         }
     }
 
+    private static final TypeReference<Map<String, Object>> DICT
+            = new TypeReference<Map<String, Object>>() { };
+    public static Map<String, Object> getDefinition(DevManager manager, String organization, String project, Integer id) {
+        HttpRequest request = new HttpRequest(
+                HttpMethod.GET,
+                String.format("https://dev.azure.com/%s/%s/_apis/build/definitions/%s?api-version=7.2-preview", organization, project, id.toString()));
+        request.setHeader(HttpHeaderName.CONTENT_TYPE, "application/json");
+        HttpResponse response = manager.serviceClient().getHttpPipeline().send(request).block();
+        System.out.println("response status code: " + response.getStatusCode());
+        if (response.getStatusCode() == 200) {
+            String body = response.getBodyAsString().block();
+            response.close();
+            try {
+                return SERIALIZER_ADAPTER.deserialize(body, DICT.getJavaType(), SerializerEncoding.JSON);
+            } catch (IOException e) {
+                throw new UncheckedIOException(e);
+            }
+        } else {
+            System.out.println("response body: " + response.getBodyAsString().block());
+            response.close();
+
+            throw new IllegalStateException("failed to get definition: " + id);
+        }
+    }
+
+    public static boolean isDefinitionEnabled(Map<String, Object> definition) {
+        return ("enabled".equalsIgnoreCase(definition.get("queueStatus").toString()));
+    }
+
+    public static void enableDefinition(DevManager manager, String organization, String project, Integer id, Map<String, Object> definition) {
+        definition.put("queueStatus", "enabled");
+        HttpRequest request = new HttpRequest(
+                HttpMethod.PUT,
+                String.format("https://dev.azure.com/%s/%s/_apis/build/definitions/%s?api-version=7.2-preview", organization, project, id.toString()));
+        request.setHeader(HttpHeaderName.CONTENT_TYPE, "application/json");
+        try {
+            request.setBody(SERIALIZER_ADAPTER.serialize(definition, SerializerEncoding.JSON));
+        } catch (IOException e) {
+            throw new UncheckedIOException(e);
+        }
+        HttpResponse response = manager.serviceClient().getHttpPipeline().send(request).block();
+        System.out.println("response status code: " + response.getStatusCode());
+        if (response.getStatusCode() == 200) {
+            response.close();
+        } else {
+            System.out.println("response body: " + response.getBodyAsString().block());
+            response.close();
+
+            throw new IllegalStateException("failed to update definition: " + id);
+        }
+    }
```

Comment:
Here's the problem of PUT.

It won't work via model class, likely due to some mismatch of JSON and model.

Therefore I had to do something akin to BinaryData and update one property in it.